src/server/pkg/obj/obj.go

```diff
@@ -66,7 +76,13 @@ func (b *BackoffReadCloser) Read(data []byte) (int, error) {
 			return err
 		}
 		return nil
-	}, b.backoffConfig)
+	}, b.backoffConfig, func(err error, d time.Duration) {
+		protolion.Debugf("%v", RetryError{
+			err:               err,
+			timeTillNextRetry: d,
+			bytesProcessed:    bytesRead,
+		})
+	})
```

Comment:
You should test this but I think by doing `Debugf` here you'll make it so these don't show up when users run the code since the debug level is set at `Info`. `Infof` will show up though.